src/qibochem/ansatz/qeb.py

```diff
@@ -0,0 +1,70 @@
+import numpy as np
+import qibo
+from qibo import Circuit, gates, models
+
+
+def qeb_circuit(n_qubits, excitation, theta=0.0, trotter_steps=1) -> Circuit:
```

Comment:
Does this ansatz involve Trotterization? If not, can take the argument out

```suggestion
def qeb_circuit(n_qubits, excitation, theta=0.0) -> Circuit:
```

Reply:
It works as a substitute for the UCC. So yes, it involves trotterization.

Reply:
Hmm, but I don't see it being used in the function right now? If it's the same as the original UCC ansatz, then the code probably needs an outside for loop to create a copy of the circuit for each Trotter step, and combine and return everything together.

Reply:
The QEB gate for any n-tuple is just basically a depth-reduced implementation of the UCC n-tuple gate. It should be directly substituted if one prefers a depth-reduced trotterized UCC. 

My concern is - if one were to use this with trotter-steps > 1, one would have to rewrite code for the outside for loop to create a circuit copy for each Trotter step, combine etc. etc. just to accommodate a simple n-tuple excitation gate substitution. By leaving it in, it allows for QEB to be plugged in directly, no? Maybe using an if-else fork at some point?